dockerfiles/ubuntu/is/README.md

```diff
@@ -19,9 +18,7 @@ git clone https://github.com/wso2/docker-is.git
 
 >The local copy of the `dockerfiles/ubuntu/is` directory will be referred to as `IS_DOCKERFILE_HOME` from this point onwards.
 
-##### 2. Copy the `wso2is-6.0.0.zip` to the `docker-is/dockerfiles/ubuntu/is` directory
-
-##### 3.  Build the Docker image.
```

Comment:
add a note saying we need to provide hosted IS 6.0.0 URL as `WSO2_SERVER_DIST_URL`